modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/experiments/Variation.kt

```diff
@@ -2,13 +2,5 @@ package au.com.shiftyjelly.pocketcasts.analytics.experiments
 
 sealed class Variation {
     data object Control : Variation()
-    data object Treatment : Variation()
-
-    companion object {
-        private const val TREATMENT = "treatment"
-        fun fromName(name: String) = when (name) {
-            TREATMENT -> Treatment
-            else -> Control
-        }
-    }
+    data class Treatment(val name: String? = null) : Variation()
 }
```

Comment:
I don't have a strong opinion about this. I was considering having this class act as a bridge so that if we ever change the API from ExPlat to another, we would only need to update the code in a few places rather than in all the experiments

Reply:
Sounds good to me to leave it too 👍 I don't predict this api to ever change, but if having this additional layer is preferable, I'm completely ok with it!